helm/templates/raw-spans-grouper-config.yaml

```diff
@@ -6,42 +6,30 @@ metadata:
     release: {{ .Release.Name }}
 data:
   application.conf: |-
-    flink.job {
-      {{- if hasKey .Values.rawSpansGrouperConfig.flink "job" }}
-      {{- if hasKey .Values.rawSpansGrouperConfig.flink.job "parallelism" }}
-      parallelism = {{ int .Values.rawSpansGrouperConfig.flink.job.parallelism }}
-      {{- end }}
-      {{- if hasKey .Values.rawSpansGrouperConfig.flink.job "metrics" }}
-      metrics {
-        metrics {
-          reporters = {{- toJson .Values.rawSpansGrouperConfig.flink.job.metrics.reporters | trim | nindent 12 }}
-        }
-      }
-      {{- end }}
-      {{- end }}
-    }
 
-    flink.source {
-      schema.registry {
-        schema.registry.url = "{{ .Values.rawSpansGrouperConfig.flink.source.schema.registry.url }}"
-        specific.avro.reader = true
-      }
-      kafka {
-        bootstrap.servers = "{{ .Values.rawSpansGrouperConfig.flink.source.kafka.bootstrapServers }}"
-      }
-    }
+    kafka.streams.config = {
+      application.id = raw-spans-to-structured-traces-grouping-job
+      state.dir = "/var/data/"
```

Comment:
Does this create a one if doesn't exist?

Reply:
yes